File: app/src/main/java/huce/fit/appreadstories/activity/GuideActivity.java
package huce.fit.appreadstories.activity;

import android.os.Bundle;
import android.widget.ImageView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import huce.fit.appreadstories.R;

public class GuideActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_guide);

        ImageView ivBack = findViewById(R.id.ivBack);
        ivBack.setOnClickListener(v -> finish());
    }
}

File: app/src/main/java/huce/fit/appreadstories/guile/GuideActivity.kt
package huce.fit.appreadstories.guile

import android.os.Bundle
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import huce.fit.appreadstories.R

class GuideActivity: AppCompatActivity() {

    private lateinit var ivBack: ImageView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_guide)
        init()
        processEvent()
    }

    private fun init() {
        ivBack = findViewById(R.id.ivBack)
    }

    private fun processEvent() {
        ivBack.setOnClickListener { finish() }
    }
}
